config/staging/jenkins.patch

```diff
@@ -256,51 +256,12 @@ index 2cec9ac..a8c78ed 100644
 +  <jenkinsUrl>http://mm-ci-staging.qa.scl3.mozilla.com:8080/</jenkinsUrl>
  </jenkins.model.JenkinsLocationConfiguration>
 \ No newline at end of file
-diff --git a/jenkins-master/jobs/mozilla-aurora_addons/config.xml b/jenkins-master/jobs/mozilla-aurora_addons/config.xml
-index af17fdd..2a65fe7 100644
---- a/jenkins-master/jobs/mozilla-aurora_addons/config.xml
-+++ b/jenkins-master/jobs/mozilla-aurora_addons/config.xml
-@@ -4,7 +4,7 @@
-   <description>Execute add-ons tests for Aurora builds.</description>
-   <logRotator class="hudson.tasks.LogRotator">
-     <daysToKeep>-1</daysToKeep>
--    <numToKeep>10</numToKeep>
-+    <numToKeep>50</numToKeep>
-     <artifactDaysToKeep>-1</artifactDaysToKeep>
-     <artifactNumToKeep>-1</artifactNumToKeep>
-   </logRotator>
-diff --git a/jenkins-master/jobs/mozilla-aurora_endurance/config.xml b/jenkins-master/jobs/mozilla-aurora_endurance/config.xml
-index 67d3f18..c897efc 100644
---- a/jenkins-master/jobs/mozilla-aurora_endurance/config.xml
-+++ b/jenkins-master/jobs/mozilla-aurora_endurance/config.xml
-@@ -4,7 +4,7 @@
-   <description>Execute endurance tests for Aurora builds.</description>
-   <logRotator class="hudson.tasks.LogRotator">
-     <daysToKeep>-1</daysToKeep>
--    <numToKeep>10</numToKeep>
-+    <numToKeep>50</numToKeep>
-     <artifactDaysToKeep>-1</artifactDaysToKeep>
-     <artifactNumToKeep>-1</artifactNumToKeep>
-   </logRotator>
 diff --git a/jenkins-master/jobs/mozilla-aurora_functional/config.xml b/jenkins-master/jobs/mozilla-aurora_functional/config.xml
-index 056de65..149a611 100644
+index 5feaa99..a9afa87 100644
 --- a/jenkins-master/jobs/mozilla-aurora_functional/config.xml
 +++ b/jenkins-master/jobs/mozilla-aurora_functional/config.xml
 @@ -4,7 +4,7 @@
-   <description>Execute functional tests for Aurora builds.</description>
-   <logRotator class="hudson.tasks.LogRotator">
-     <daysToKeep>-1</daysToKeep>
--    <numToKeep>10</numToKeep>
-+    <numToKeep>50</numToKeep>
-     <artifactDaysToKeep>-1</artifactDaysToKeep>
-     <artifactNumToKeep>-1</artifactNumToKeep>
-   </logRotator>
-diff --git a/jenkins-master/jobs/mozilla-aurora_l10n/config.xml b/jenkins-master/jobs/mozilla-aurora_l10n/config.xml
-index fd615c3..387e40c 100644
---- a/jenkins-master/jobs/mozilla-aurora_l10n/config.xml
-+++ b/jenkins-master/jobs/mozilla-aurora_l10n/config.xml
-@@ -4,7 +4,7 @@
-   <description>Execute l10n tests for Aurora builds.</description>
+   <description>Execute functional tests for Firefox Developer Edition builds.</description>
    <logRotator class="hudson.tasks.LogRotator">
      <daysToKeep>-1</daysToKeep>
 -    <numToKeep>10</numToKeep>
```

Comment:
We do no longer run l10n tests for aurora. We can restart with those once we have a better strategy on them. As result I removed the job from the config.